src/ucp/tag/offload.c

```diff
@@ -428,12 +431,20 @@ ucs_status_t ucp_tag_offload_rndv_zcopy(uct_pending_req_t *self)
     size_t max_iov     = ucp_ep_config(ep)->tag.eager.max_iov;
     uct_iov_t *iov     = ucs_alloca(max_iov * sizeof(uct_iov_t));
     size_t iovcnt      = 0;
+    ucp_rsc_index_t md_index;
     ucp_dt_state_t dt_state;
     void *rndv_op;
 
+    md_index = ucp_ep_md_index(ep, req->send.lane);
+
+    UCS_STATIC_ASSERT(ucs_is_pow2(UCS_SYS_CACHE_LINE_SIZE));
+
+    ucs_assert_always(md_index < UCS_SYS_CACHE_LINE_SIZE);
+    ucs_assert_always(!((uintptr_t)req & (UCS_SYS_CACHE_LINE_SIZE - 1)));
```

Comment:
maybe add `#define UCP_TAG_OFFLOAD_REQPTR_MDI_MASK  (UCS_SYS_CACHE_LINE_SIZE - 1)`
and use it in all places, including the assertions?

Reply:
done